coverage/src/main/java/com/github/lizardev/xquery/saxon/coverage/report/StaticResourceTransferor.java

```diff
@@ -2,23 +2,33 @@
 
 import java.io.File;
 import java.io.IOException;
+import java.io.InputStream;
 import java.net.URISyntaxException;
 
 import static org.apache.commons.io.FileUtils.copyFileToDirectory;
+import static org.apache.commons.io.FileUtils.copyInputStreamToFile;
 
 public class StaticResourceTransferor {
 
     private static final String BASE_PATH = "com/github/lizardev/xquery/saxon/coverage/report";
 
     public void copyResources(File baseDir) {
```

Comment:
i would leave only one function for coping resources, e.g. copyResource(resourcePath, File destFile)